src/appleseed/renderer/kernel/lighting/lightsample.h

```diff
@@ -97,7 +97,7 @@ class LightSample
 
     // Data for a light-emitting shape sample.
     const EmittingShape*        m_shape;
-    foundation::Vector2f        m_bary;                         // barycentric coordinates of the sample
+    foundation::Vector2f        m_param;                        // parametric coordinates of the sample

```

Comment:
As suggested on Discord, `m_parametric` may be less confusing since we frequently use `m_params` for entity parameters.